addons/sourcemod/scripting/AutoSilencer.sp

```diff
@@ -0,0 +1,169 @@
+#include <sourcemod>
+#include <sdktools>
+#include <sdkhooks>
+#include <clientprefs>
+#include <multicolors>
+
+#pragma semicolon 1
+#pragma newdecls required
+
+bool primary[2048];
+bool g_bIsAutoSEnabled[MAXPLAYERS + 1] = {false, ...};
+Handle g_hCookie = INVALID_HANDLE;
+
+public Plugin myinfo =
+{
+	name = "SM Autosilencer",
+	author = "Franc1sco franug, Dolly, .Rushaway",
```

Comment:
where did you find this plugin? (we can fork it if it exists on the original author github)

Reply:
https://github.com/Franc1sco/AutoSilencer/tree/master

Reply:
I will wait for you ping when you forked the repo, then I will open this PR again.

Reply:
done here:
- https://github.com/srcdslab/sm-plugin-autosilencer

Reply:
@Rushaway

Reply:
https://github.com/srcdslab/sm-plugin-autosilencer/pull/1